src/main/fc/core.c

```diff
@@ -462,19 +469,12 @@ void disarm(flightLogDisarmReason_e reason)
         UNUSED(reason);
 #endif
         BEEP_OFF;
-#ifdef USE_DSHOT
-        if (isMotorProtocolDshot() && flipOverAfterCrashActive && !featureIsEnabled(FEATURE_3D)) {
-            dshotCommandWrite(ALL_MOTORS, getMotorCount(), DSHOT_CMD_SPIN_DIRECTION_NORMAL, DSHOT_CMD_TYPE_INLINE);
```

Comment:
Without this, motors may stay reversed when disarmed. Is that safe?

Reply:
Yes, to be honest I was expecting something bad to happen, but it didn't.  The motors are set to off anyway, and on arm, they get set to the appropriate direction.  It doesn't seem to matter what the previous direction command was.  For example, if the turtle mode switch is on, and then we arm after normal flight, the preceding direction was 'normal', and only at arm time is it 'reversed'.  But they all seem to start OK, like that.  From testing, it appears unnecessary, and may originally have been put there 'just in case'.

Reply:
> Yes, to be honest I was expecting something bad to happen, but it didn't. 

_Yet_ ;-)

Personally I prefer 'just in case'. It will at least slightly reduce size of quad state when disarmed. And, with unconditional setting on ARM, simply always return DSHOT motors to normal.

Reply:
If we need to refactor to only send the DShot direction commands once, on the transitions, then I think it will be absolutely necessary to include those lines.  
I just don't know, for sure, if the direction commands are best sent 'all the time' or just 'once only'.